lib/simulator-xcode-11.4.js

```diff
@@ -104,6 +105,25 @@ class SimulatorXcode11_4 extends SimulatorXcode11 {
   async clearKeychains () {
     await this.simctl.resetKeychain();
   }
+
+  /**
+   * Opens simulator's UI Client if not already open and boots sim
+   *
+   * @param uiClientPid - process id of simulator UI client.
+   * @param opts - arguments to start simulator UI client with.
+   * @returns {Promise} - a promise that resolves when sim UI client is launched and sim is booted.
+   * @override
+   */
+  async launchSimWindow (uiClientPid, opts) {
```

Comment:
I would rather change the `uiClientPid` argument to boolean `isUiClientRunning`

Reply:
Makes sense, I changed it.